conda_smithy/configure_feedstock.py

```diff
@@ -292,7 +298,11 @@ def copytree(src, dst, ignore=(), root_dst=None):
 def copy_feedstock_content(forge_dir):
     feedstock_content = os.path.join(conda_forge_content,
                                      'feedstock_content')
-    copytree(feedstock_content, forge_dir, 'README')
+    copytree(
+        feedstock_content,
+        forge_dir,
+        ['README', 'ci_support/disabled.svg']
```

Comment:
Was also concerned that it might be copying it to the feedstock still a couple weeks back. Adding this line ensures the badge is not copied to the feedstock. Have already tested that it works.